### ISSUE TITLE
Disclose ARKs in HTTP response headers

### ISSUE THREAD

**Issue by machawk1** [metadata-research/yamz#151]
...or even the HTML headers. @hvdsomp might have two cents on this.

**Comment by hvdsomp** [comment]
I am not sure what the question/issue is. But if this is about pointing from a landing page to the persistent identifier of the object described by that landing page, then there's the "cite-as" link relation type that was specifically defined for that purpose in [RFC8574](https://www.rfc-editor.org/rfc/rfc8574.html). The target of the link must be a URI, preferably an HTTP URI.

**Comment by machawk1** [comment]
@hvdsomp It's something a little different. We have a PID (ARK) that is embedded in the body of a web page. The GH issue here is whether we should facilitate the web page effectively "announcing" its ARK via either HTML headers or HTTP response headers. This would be in a similar vein of a paper's landing page in a DL making its DOI machine-readable. Thoughts?

**Comment by jkunze** [comment]
Yes, I agree that the term+definition pairs in yamz.net should disclose cite-as URIs in response headers.
What I'm unsure about is if it makes sense to have one cite-as URI for the term+definition in its current (frozen) state and another pseudo-cite-as URI for the term+definition in its latest (living) state. The latter is a common thing to want to reference. What's a good way to declare it?
